src/main/java/dk/kb/storage/api/v1/impl/DsStorageApiServiceImpl.java

```diff
@@ -270,16 +270,25 @@ public  List<OriginCountDto> getOriginStatistics() {
         }
     }
 
-	@Override
-	public Integer deleteRecordsForOrigin(String origin, Long mTimeFrom, Long mTimeTo) {
-		   try {
-	            log.debug("deleteRecordsForOrigin() called with call details: {}", getCallDetails());
-	            return DsStorageFacade.deleteRecordsForOrigin(origin,mTimeFrom,mTimeTo);
-	        } catch (Exception e) {
-	            throw handleException(e);
-	        }
-
-	}
+    @Override
+    public Integer deleteRecordsForOrigin(String origin, Long mTimeFrom, Long mTimeTo) {
+        try {
+	    log.debug("deleteRecordsForOrigin() called with call details: {}", getCallDetails());
+	    return DsStorageFacade.deleteRecordsForOrigin(origin,mTimeFrom,mTimeTo);
+        } catch (Exception e) {
+	    throw handleException(e);
+        }
+    }
+    
+    @Override
+    public void updateKalturaId(String kalturaReferenceId, String kalturaInternalId) {
+        try {
+            log.debug("updateKalturaId() called with call details: {}", getCallDetails());
+            //return DsStorageFacade.               
+        } catch (Exception e) {
+            throw handleException(e);
+                }
+            }
```

Comment:
This method doesn't do anything. Is that because it haven't been implemented yet?

Reply:
Is called facade now